app/controllers/api/users_controller.rb

```diff
@@ -3,6 +3,24 @@ module Api
   class UsersController < ApplicationController
     include Devise::Controllers::Helpers
 
+    def show
```

Comment:
Can a guest (logged out user) access this endpoint?
Let's secure this, only logged in users should be allowed to get details about a golfer's scores.